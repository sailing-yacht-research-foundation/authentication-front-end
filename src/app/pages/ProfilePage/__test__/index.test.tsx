
import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { ProfilePage } from '..';
import MyProvider from 'app/components/Provider';

const shallowRenderer = createRenderer();

describe('<ProfilePage />', () => {
  it('should render to match the snapshot', () => {
    shallowRenderer.render(
      <MyProvider>
        <ProfilePage />
      </MyProvider>);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});