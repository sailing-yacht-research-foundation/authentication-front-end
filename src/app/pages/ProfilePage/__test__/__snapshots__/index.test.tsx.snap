// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfilePage /> should render to match the snapshot 1`] = `
<Provider
  store={
    Object {
      "createReducer": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "injectedReducers": Object {},
      "injectedSagas": Object {},
      "replaceReducer": [Function],
      "runSaga": [Function],
      "subscribe": [Function],
      Symbol(Symbol.observable): [Function],
    }
  }
>
  <ProfilePage />
</Provider>
`;
